azkaban-common/src/main/java/azkaban/jobExecutor/JavaProcessJob.java

```diff
@@ -48,6 +50,7 @@ public class JavaProcessJob extends ProcessJob {
   public static final String NATIVE_LIBRARY_PATH_PREFIX = "native.library.path.";
 
   public static String JAVA_COMMAND = "java";
+  public static String DEFAULT_JAVA11_COMMAND = "java11";
```

Comment:
Just curious: in what case the job is gonna use a non-default Java11 cmd (when java11 is enabled)? And what else the Java11 cmd could be?

Reply:
I also want to check this default java11, will we create ENV that reflect java11 into a workable java11-msft in pod? 
And do we allow user to custom pass a path_to_java and allow them to install this custom version in the pod?

Reply:
Good point. I just changed the getJobProps().getString(JAVA11_COMMAND_PARAMS) to getSysProps().getString(JAVA11_COMMAND_PARAMS). In that way, user cannot modify that config value by themselves. Instead, it will always be the value coming from our commonprivate.properties

Reply:
hi ! Does DEFAULT_JAVA11_COMMAND indicate the binary path ? 
This does assume we will map the java11 sdk where ever it lies to 'java11' right ? 

Also,  do you think JAVA11_COMMAND_PARAMS can be renamed to JAVA11_BINARY_PATH

and. RUN_ON_JAVA11_PARAMS to JAVA11_JVM_PARAMS . it seems a little clearer to me from a naming convention perspective.

Reply:
> Also, do you think JAVA11_COMMAND_PARAMS can be renamed to JAVA11_BINARY_PATH & RUN_ON_JAVA11_PARAMS to JAVA11_JVM_PARAMS

Agreed. I also think the naming is confusing...

> This does assume we will map the java11 sdk where ever it lies to 'java11' right ?

Correct. But I think using JAVA11_COMMAND_PARAMS is better to specify the Java11 binary path. not like python and python3, i didn't see java8 and java11 are named java8 and java11 as the binary name. Typically both are named java. Using binary path is probably a better way to distinguish them.

Reply:
> I also want to check this default java11, will we create ENV that reflect java11 into a workable java11-msft in pod?

I think we can use JAVA11_BINARY_PATH to specify the java11 path. It is more appropriate i think.